zabbix/repo.sls

```diff
@@ -24,13 +24,14 @@
     - name: apt-transport-https
 {{ id_prefix }}_repo:
   pkgrepo.managed:
-    - name: deb https://repo.zabbix.com/zabbix/{{ zabbix.version_repo }}/{{ salt['grains.get']('os')|lower }} {{ salt['grains.get']('oscodename') }} main
+    - name: deb [ arch={{ salt['grains.get']('osarch')|lower }} {{ zabbix.repo_signedby }} ] https://repo.zabbix.com/zabbix/{{ zabbix.version_repo }}/{{ salt['grains.get']('os')|lower }} {{ salt['grains.get']('oscodename') }} main
     - file: /etc/apt/sources.list.d/zabbix.list
     - key_url: https://repo.zabbix.com/zabbix-official-repo.key
     - clean_file: True
+    - aptkey: {{ zabbix.repo_aptkey }}
 
 {%- elif salt['grains.get']('os_family') == 'RedHat' and
-         salt['grains.get']('osmajorrelease')|int >= 6 %}
+        salt['grains.get']('osmajorrelease')|int >= 6 %}
```

Comment:
Please don't add unnecessary  changes. No reason to change indentation in this line.

Reply:
My linter told me to do that so I did it.